help.go

```diff
@@ -10,13 +10,13 @@ const defaultHelpTemplate = `{{.CommandName}}{{if .Description}} - {{.Descriptio
     {{.UsageString}}{{end}}{{if .Positionals}}
 
   Positional Variables: {{range .Positionals}}
-    {{.Name}}{{if .Required}} (Required){{end}}{{if .Description}} - {{.Description}}{{end}}{{if .DefaultValue}} (default: {{.DefaultValue}}){{end}}{{end}}{{end}}{{if .Subcommands}}
+    {{.Name}}  {{.Spacer}}{{if .Description}} {{.Description}}{{end}}{{if .DefaultValue}} (default: {{.DefaultValue}}){{else}}{{if .Required}} (Required){{end}}{{end}}{{end}}{{end}}{{if .Subcommands}}
```

Comment:
Beside the spacer, I moved the required flag at the end. It now is only shown if there is no default value. Having a default value makes the requirement superfluous.

Reply:
Really good logic.  I like it!